content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -881,24 +911,20 @@ quorum read to be served.
 | resourceVersionMatch param            | paging params                 | resourceVersion unset | resourceVersion="0"                       | resourceVersion="{value other than 0}" |
```

Comment:
If you want to, you can add a [table caption](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#table-captions)

Reply:
Hi @tengqm . The changes look good. One question:
Have a look line 933,
https://deploy-preview-23669--kubernetes-io-master-staging.netlify.app/docs/reference/using-api/api-concepts/#unavailable-resource-versions
A footnote (?) appears at the bottom of the rendered page before the "feedback" section.
`If the server does not honor the resourceVersionMatch parameter, it is treated as if it is unset`

Reply:
Yes. I don't like the way it is rendered. Maybe we should avoid using the footnote syntax? Any suggestions?

Reply:
Pls check the latest commit, I'm simulating the footnote so that it renders immediately after the table.